<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestionEmploye</class>
 <widget class="QMainWindow" name="GestionEmploye">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1684</width>
    <height>881</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1631</width>
      <height>811</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(243, 217, 255);
font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Ajouter Employé</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>241</width>
        <height>721</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="title">
       <string>Ajout</string>
      </property>
      <widget class="QPushButton" name="pb_ajouter">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>640</y>
         <width>61</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/image/image/ajouter.png</normaloff>:/image/image/ajouter.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_supprimer">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>640</y>
         <width>61</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/image/image/delet.png</normaloff>:/image/image/delet.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_modifier">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>640</y>
         <width>61</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/image/image/edit.png</normaloff>:/image/image/edit.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>160</width>
         <height>571</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Identifiant</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_id"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>N° CIN</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_cin"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Prenom</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_prenom"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Nom</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_nom"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Salaire</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_salaire"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Email</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_email"/>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_14">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_10">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Login</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="le_login"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_11">
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>mot de passe</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="le_pwd"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_12">
                <item>
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string>Profile</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="cb_profile">
                  <item>
                   <property name="text">
                    <string>Administrateur</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Personel</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Directeur</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_13">
                <item>
                 <widget class="QLabel" name="label_13">
                  <property name="text">
                   <string>Date de naissance</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDateEdit" name="dateEdit"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QTableView" name="tab_emp">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>60</y>
        <width>1281</width>
        <height>511</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_recherche">
      <property name="geometry">
       <rect>
        <x>1480</x>
        <y>10</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/image/image/recherche.png</normaloff>:/image/image/recherche.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_recherche">
      <property name="geometry">
       <rect>
        <x>1220</x>
        <y>10</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_trierNom">
      <property name="geometry">
       <rect>
        <x>1140</x>
        <y>10</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/image/image/trie.png</normaloff>:/image/image/trie.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_trie">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>10</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>nom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>prenom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>salaire</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pb_refresh">
      <property name="geometry">
       <rect>
        <x>1560</x>
        <y>10</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/image/image/refresh.png</normaloff>:/image/image/refresh.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_stat_2">
      <property name="geometry">
       <rect>
        <x>1500</x>
        <y>600</y>
        <width>101</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/image/image/chatbox.png</normaloff>:/image/image/chatbox.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>95</width>
        <height>100</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_pdf">
      <property name="geometry">
       <rect>
        <x>1370</x>
        <y>600</y>
        <width>101</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/image/image/pdf.png</normaloff>:/image/image/pdf.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_stat">
      <property name="geometry">
       <rect>
        <x>1240</x>
        <y>600</y>
        <width>101</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/image/image/statistic.png</normaloff>:/image/image/statistic.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>To Do List</string>
     </attribute>
     <widget class="QTableView" name="tab_tache">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>220</y>
        <width>411</width>
        <height>381</height>
       </rect>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>220</y>
        <width>291</width>
        <height>391</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="title">
       <string>Description du tâche</string>
      </property>
      <widget class="QPushButton" name="pb_ajoutTache">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>300</y>
         <width>61</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/image/image/ajouter.png</normaloff>:/image/image/ajouter.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
      <widget class="QLineEdit" name="des_tache">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>261</width>
         <height>141</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="etat_tache">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>261</width>
         <height>41</height>
        </rect>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>A FAIRE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>EN COURS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FAIT</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pb_modifTache">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>300</y>
         <width>61</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/image/image/edit.png</normaloff>:/image/image/edit.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>130</y>
        <width>411</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pb_afficherDrct">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>20</y>
         <width>61</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/image/image/afficher.png</normaloff>:/image/image/afficher.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_idEmp">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_id_3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>53</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>ID_emp</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pb_refresh_2">
      <property name="geometry">
       <rect>
        <x>1170</x>
        <y>20</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/image/image/refresh.png</normaloff>:/image/image/refresh.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_profileEmp">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>20</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_afficherTache">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>20</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/image/image/valider.png</normaloff>:/image/image/valider.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>35</width>
        <height>40</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_id">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>20</y>
        <width>16</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_id_2">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>20</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>311</width>
        <height>411</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/image/image/toDoList.png</normaloff>:/image/image/toDoList.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>400</width>
        <height>400</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>20</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/image/image/profile.png</normaloff>:/image/image/profile.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QGroupBox" name="gb_delet">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>610</y>
        <width>171</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pb_suppTache">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>10</y>
         <width>61</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/image/image/delet.png</normaloff>:/image/image/delet.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_idTache">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QComboBox" name="cb_etat">
      <property name="geometry">
       <rect>
        <x>1180</x>
        <y>220</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>TOUT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>A FAIRE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>EN COURS</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>FAIT</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pb_fitre">
      <property name="geometry">
       <rect>
        <x>1310</x>
        <y>220</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Filtre</string>
      </property>
      <property name="iconSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Arduino</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>160</y>
        <width>401</width>
        <height>301</height>
       </rect>
      </property>
      <property name="title">
       <string>Pointage</string>
      </property>
      <widget class="QPushButton" name="pb_arduino">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>100</y>
         <width>101</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/image/image/arduino.png</normaloff>:/image/image/arduino.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>125</width>
         <height>125</height>
        </size>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_Arduino">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>130</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLineEdit" name="pointage">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>480</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1684</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
